src/async_fuse/memfs/kv_engine/mod.rs

```diff
@@ -225,8 +232,24 @@ pub trait KVEngine: Send + Sync + Debug + Sized {
     /// Lease grant
     async fn lease_grant(&self, ttl: i64) -> DatenLordResult<i64>;
 
+    /// Revoke a lease
+    async fn lease_revoke(&self, lease_id: i64) -> DatenLordResult<()>;
+
+    /// Lease keep alive
+    async fn create_keep_alive_session(
```

Comment:
The name can be simplify to `create_session()` because session is always keepalived. It would be better to move `lease_grant()` function inside to make interface cleaner.

Reply:
Ok, done.